server/chat-commands/room-settings.ts

```diff
@@ -1310,11 +1301,13 @@ export const commands: ChatCommands = {
 		const resetTier = cmd === 'resettierdisplay';
 		if (!target) {
 			if (!this.runBroadcast()) return;
-			return this.sendReplyBox(`This room is currently displaying ${room.dataCommandTierDisplay} as the tier when using /data.`);
+			return this.sendReplyBox(
+				`This room is currently displaying ${room.settings.dataCommandTierDisplay} as the tier when using /data.`
+			);
 		}
 		if (!this.can('declare', null, room)) return false;
 
-		const displayIDToName: {[k: string]: typeof room.dataCommandTierDisplay} = {
+		const displayIDToName: {[k: string]: "tiers" | "doubles tiers" | "numbers"} = {
```

Comment:
```suggestion
		const displayIDToName: {[k: string]: typeof room.settings.dataCommandTierDisplay} = {
```

The `typeof` is important! It makes the code DRYer: DRY stands for Don't Repeat Yourself, and making the code DRY means that if we ever add more settings, we only have to change it in one place instead of multiple places.